docs/usage-as-end-user.md

```diff
@@ -156,6 +156,62 @@ spec:
       enabled: true
 ```
 
+Please find below an example `Shoot` manifest to use GCP's local SSD:
```

Comment:
Thanks for the example. I was also more thinking about a small section similar to https://github.com/gardener/gardener-extension-provider-gcp/blob/master/docs/usage-as-end-user.md#infrastructureconfig or https://github.com/gardener/gardener-extension-provider-gcp/blob/master/docs/usage-as-end-user.md#controlplaneconfig where you quickly explain the `WorkerConfig`.

Reply:
Accept